spec/unit/ordinals_spec.rb

```diff
@@ -46,3 +46,13 @@
     end
   end
 end
+
+  describe 'Occitan' do
```

Comment:
It looks to me like these lines should be *inside* `describe 'Ordinals for' do`.

Please try and make the specs work locally 🙏🏻